src/Hooks/useAuthForm.jsx

```diff
@@ -0,0 +1,40 @@
+import { useReducer } from "react";
```

Comment:
this could have been kept in a context file instead of placing it in a custom hook file

Reply:
useAuthForm doesn't create a context, it just creates and maintains the state for the Authentication form inputs. My options were either to create a separate file for the reducer in the components folder or to keep the logic in one place for better readability,  so I went with this, as didn't want to create nested folders and this made sense at the time.